config/spree_avatax_config.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+require_relative '../app/models/spree_avatax_official/configuration'
+require_relative '../app/models/spree_avatax_official/calculator/avatax_transaction_calculator'
+
+Rails.application.config.spree = Spree::Core::Environment.new
+SpreeAvataxOfficial::Config    = SpreeAvataxOfficial::Configuration.new
```

Comment:
is this ready right now if you moved configuration to hw-admin ?

Reply:
Hmm :thinking: I thinks we can remove that, checked locally and it works fine, also no deprecation warnings